book_maker/cli.py

```diff
@@ -423,7 +433,7 @@ def main():
         if not options.api_base:
             raise ValueError("`api_base` must be provided when using `deployment_id`")
         e.translate_model.set_deployment_id(options.deployment_id)
-    if options.model == "openai":
+    if options.model in ("openai", "groq"):
```

Comment:
this seems wrong it always raise error

Reply:
Maybe you need run with --model_list arg.

Reply:
Because Groq needs to specify the model, I reuse the way of specifying the model.